<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QStackedWidget" name="GuberWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>841</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>580</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userPassword">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>361</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="userName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>361</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="enterName">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterPassword">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Password:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bookingChoice">
     <widget class="QPushButton" name="carBookingButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Car Booking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="planeBookingButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>260</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Plane booking</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="carBookingPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>CAR BOOKING PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmBookingCar">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>490</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm booking</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="planeBookingPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PLANE BOOKING PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmBookingPlane">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>510</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Booking</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="confirmationPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CONFIRMATION PAGE / TICKET PAGE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
